apps/crawler/src/id_crawler.mjs

```diff
@@ -140,24 +143,24 @@ async function extractDownloadLinks(html, downloadDir) {
         const remoteFileHash = await calculateRemoteFileHash(fullUrl);
 
         if (remoteFileHash && existingFileHash === remoteFileHash) {
-          console.log(
+          logger.debug(
```

Comment:
@HixDr I believe this should be at 'info' level. You already have 'info' level at elseif statement. Why did you choose 'debug' in this?